### PR TITLE
add instructions for development with Docker without VS Code

### PR DESCRIPTION
**Pull request by eeshugerman**
NOTE: lines 57 and 58 assume changes from https://github.com/Requarks/wiki/pull/1905

### CODE REVIEW

File: dev.md [hunk 1]
```diff
@@ -17,10 +16,10 @@ There are 2 methods to develop for Wiki.js. You can either use the dockerized de
 * Docker
 * Docker Compose
 * Linux / macOS / Windows 10 Pro or Enterprise
-* Visual Studio Code
 
-## Run the project
+## Visual Studio Code
 
+### Running the project
```
Comment (eeshugerman) [inline dev.md:23]: Changed "Run" to "Running" to match the tense of the next two headings

**Comment by eeshugerman** [comment]
@NGPixel just wanted to make sure you saw this

File: dev.md [hunk 2]
```diff
@@ -39,15 +39,42 @@ There are 2 methods to develop for Wiki.js. You can either use the dockerized de
...
 
-An alternate method is to open a command prompt in the `dev/containers` folder and run `docker-compose down`.
+## Generic
+
+### Running the project
+1. Clone the project from [GitHub](https://github.com/Requarks/wiki).
+2. Run the following commands:
+```
+docker-compose -f dev/containers/docker-compose.yml up -d
+docker exec wiki-app yarn   # only necessary the first time (todo: why is it necessary at all?)
```
Comment (NGPixel) [inline dev.md]: In order to install the npm dependencies.

Comment (eeshugerman) [reply dev.md]: Ah got it, I see that step now under "Manual". I'll remove the todo. I guess VS Code takes care of this step automatically?

Comment (NGPixel) [reply dev.md]: Correct, see https://github.com/Requarks/wiki/blob/2fe2e6210da79003ac229027dfb06a9f992f47fd/.devcontainer/devcontainer.json#L32

**Review by NGPixel — APPROVED** [review]

**Pull request merged**